### ISSUE TITLE
Issue on PSNR

### ISSUE THREAD

**Issue by zoharli** [ShenghaiRong/caffe_srgan#2]
Hi , glad to see you reproduced the similar result as presented in the paper. I also implemented srResNet-mse in tensorflow , and don't know why I can't reproduce the PSNR as presented. 
I used RGB format to train the network and do PSNR assessment, and got an average PSNR of around 24dB ,then I converted the output to Y-channel using the formula  y = 16 + 0.183*r + 0.614*g + 0.062*b , and get a result around 25dB. 
So can you help me on this issue? What's your input format, is it RGB or YUV?

**Comment by ShenghaiRong** [comment]
My training input format is RGB. Training set is train.h5 : 192000*3*75*75. With respect to PSNR and SSIM, "all reported PSNR and SSIM measures were caculated on the y-channel of center-cropped, removal of a 4-pixel wide strip from each border."------paper of SRGAN

**Comment by zoharli** [comment]
That's really weird, according to my observation,there are several other implementations of SRGAN , most of them encountered the same problem as mine----low PNSR and blurred output,though we accurately implemented the network as in paper.Is there any trick that's not proposed in paper when we train the srResNet-mse network?
I'm not familiar with caffe, but I found there may be some slight differences of our implementations, did you use bias and weight parameter in  batch normlizaiton?
